src/Scripts/Game/Feature/Quantity/SCR_InventoryStorageManagerComponent.c

```diff
@@ -0,0 +1,110 @@
+	}
+
+	//------------------------------------------------------------------------------------------------
+	void EL_RequestQuantitySplit(notnull IEntity sourceEntity, int splitSize = -1)
+	{
+		Rpc(RPC_EL_RequestQuantitySplit, EL_Utils.GetRplId(sourceEntity), splitSize);
+	}
+
+	//------------------------------------------------------------------------------------------------
+	[RplRpc(RplChannel.Reliable, RplRcver.Server)]
+	protected void RPC_EL_RequestQuantitySplit(RplId quantitySourceRplId, int splitSize)
+	{
+		EL_QuantityComponent quantitySource = EL_ComponentFinder<EL_QuantityComponent>.Find(EL_Utils.FindEntityByRplId(quantitySourceRplId));
+		if (!quantitySource || !EL_CanManipulate(quantitySource.GetOwner())) return;
+		quantitySource.Split(splitSize);
+	}
+
+	//------------------------------------------------------------------------------------------------
+	void EL_RequestQuantityTransfer(notnull IEntity sourceEntity, notnull IEntity destinationEntity, int amount = -1, bool setQuantity = false)
+	{
+		Rpc(RPC_EL_QuantityTransfer, EL_Utils.GetRplId(sourceEntity), EL_Utils.GetRplId(destinationEntity), amount, setQuantity);
+	}
+
+	//------------------------------------------------------------------------------------------------
+	[RplRpc(RplChannel.Reliable, RplRcver.Server)]
+	protected void RPC_EL_QuantityTransfer(RplId quantitySourceRplId, RplId quantityDestinationRplId, int amount, bool setQuantity)
+	{
+		EL_QuantityComponent quantitySource = EL_ComponentFinder<EL_QuantityComponent>.Find(EL_Utils.FindEntityByRplId(quantitySourceRplId));
+		if (!quantitySource || !EL_CanManipulate(quantitySource.GetOwner())) return;
+
+		EL_QuantityComponent quantityDestination = EL_ComponentFinder<EL_QuantityComponent>.Find(EL_Utils.FindEntityByRplId(quantityDestinationRplId));
+		if (!quantityDestination || !EL_CanManipulate(quantityDestination.GetOwner())) return;
+
+		quantityDestination.Combine(quantitySource, amount, setQuantity);
+	}
+
+	//------------------------------------------------------------------------------------------------
+	void EL_SetTransferIntent(notnull IEntity sourceEntity, bool keepSeperate)
+	{
+		Rpc(RPC_EL_SetTransferIntent, EL_Utils.GetRplId(sourceEntity), keepSeperate);
+	}
+
+	//------------------------------------------------------------------------------------------------
+	[RplRpc(RplChannel.Reliable, RplRcver.Server)]
+	protected void RPC_EL_SetTransferIntent(RplId sourceRplId, bool keepSeperate)
+	{
+		IEntity sourceEntity = EL_Utils.FindEntityByRplId(sourceRplId);
+		if (!sourceEntity || !EL_CanManipulate(sourceEntity)) return;
+		EL_QuantityComponent.SetTransferIntent(sourceEntity, keepSeperate);
+	}
+
+	//------------------------------------------------------------------------------------------------
+	protected bool EL_CanManipulate(IEntity entity)
+	{
+		const int maxManipulationRange = 10;
+		return (vector.Distance(entity.GetOrigin(), GetOwner().GetOrigin()) < maxManipulationRange);
```

Comment:
No attribute for this?
```suggestion
		return (vector.Distance(entity.GetOrigin(), GetOwner().GetOrigin()) < 10);
```

Reply:
This is only a serverside rough validation. It could be 50 meters but for now, I eyeballed it at 10. Not really something somebody should need to adjust.